sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py

```diff
@@ -125,6 +121,7 @@ def on_error(partition_context, error):
 
 @pytest.mark.liveTest
 def test_receive_over_websocket_sync(connstr_senders):
+    pytest.skip("Waiting on uAMQP release")
```

Comment:
Nit:  Could we specify more precisely what we're waiting for? (selfishly when I was going through and removing some of these on prior tests, it would have been useful to know whether I had _actually_ crossed whatever threshold the original author intended, and am not missing some context)